test/libsolidity/syntaxTests/storageLayoutSpecifier/library.sol

```diff
@@ -0,0 +1,5 @@
+library L layout at 42 { }
+// ====
+// stopAfter: parsing
+// ----
+// ParserError 2314: (10-16): Expected '{' but got identifier
```

Comment:
You don't need `stopAfter` if it errors out.

Same in `interface.sol` and `function_declaration_layout_specified.sol`.